components/issue-list-item.tsx

```diff
@@ -67,7 +76,7 @@ export default function IssueListItem({issue = null}:{issue?: IssueData}) {
                             }
                         </h4>
                         <div className="d-flex align-center flex-wrap align-items-center justify-content-md-start">
-                            <span className={`status ${handleColorState(issue?.state)} mr-2 mt-1`}>{issue?.state}</span>
+                            <span className={`status ${handleColorState(replaceRedeem(issue?.state))} mr-2 mt-1`}>{replaceRedeem(issue?.state)}</span>
```

Comment:
Redeemed status does not exists ao this should be migrated with sqlize

Reply:
Finish